app/views/issues/_form.html.erb

```diff
@@ -15,9 +15,11 @@
         class: 'textile h-100',
         data: {
           allow_dropdown: @issue.new_record?,
-          behavior: 'rich-toolbar',
+          autosave_key: "#{current_project.id}-issue-#{ @issue.persisted? ? "edit-#{@issue.id}" : 'new' }",
```

Comment:
@brianp What's this `autosave_key` here for?

Reply:
Not sure exactly. Merge conflict? Issues should have one though yeah?

Reply:
Confirmed. It's a merge conflict and not-needed.